test/unit/init/SeededPodInitializer.test.ts

```diff
@@ -0,0 +1,46 @@
+import fs from 'fs';
+import type { RegistrationManager } from '../../../src/identity/interaction/email-password/util/RegistrationManager';
+import { SeededPodInitializer } from '../../../src/init/SeededPodInitializer';
+
+jest.mock('fs');
+const mockFs = fs as jest.Mocked<typeof fs>;
```

Comment:
```suggestion
const cache = mockFs('/');
```
Or another path. This is a utility test function we have to mock all the fs functions (that we needed so far, but `readFile` is part of it). That function returns a key/value object that simulates the file system. Because overwriting the function as you do below could have impact on other tests.

Reply:
Nice. I should use that in my other projects 👀